lib/read_parsers.cc

```diff
@@ -41,6 +41,7 @@ SeqAnParser::SeqAnParser( char const * filename ) : IParser( )
     }
     __asm__ __volatile__ ("" ::: "memory");
     _private->seqan_spin_lock = 0;
+    _num_reads = 0;
```

Comment:
Isn't this redundant with line 120? https://github.com/ged-lab/khmer/pull/877/files#diff-be310f56bb83cef45967cfbe13f0adeeR120